deployment/helm/human-connection/Chart.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+appVersion: "0.3.1"
+description: A Helm chart for Human Connection
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2020-02-04T13:00:00Z" title="Tuesday, February 4th 2020, 2:00:00 pm +01:00">Feb 4, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -1,5 +1,5 @@
 apiVersion: v1
-appVersion: "0.1.0"
+appVersion: "0.2.2"
```


```suggestion
appVersion: "0.3.0"
```
Now. We should keep that in sync somehow. Maybe with `yarn run release`?

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-02-04T13:27:44Z" title="Tuesday, February 4th 2020, 2:27:44 pm +01:00">Feb 4, 2020</time>_

---


I could try that out... actually, we had some complaints that the version number is not correct in production ... I think I didn't run it properly